### DISCUSSION TITLE
Compression support for se_atten descriptor type

### DISCUSSION THREAD

**Discussion by morethankk** [deepmodeling/deepmd-kit#2475] [Q&A] [answered]
Hello development team, I noticed that the content about descriptors supporting model compression is in the latest 2.2.1 version "Descriptors with se_e2_a, se_e3, and se_e2_r types are supported by the model compression feature. Hybrid mixed with the above descriptors is also supported.” Have you considered adding support for compression of se_atten descriptor types? Thanks a lot！

**Comment by njzjz** [comment, marked as answer]
We will track the progress in #2104.
